scripts/README.md

```diff
@@ -27,7 +27,7 @@ These scripts have been created to simplify the testing of each template within
     - Create a new directory within `STACKS-TESTING`called `test-templates`.
     - Create new templates for each existing within `stacks-dotnet`:
         - Simple.WebAPI - `stacks-webapi`
-        - CQRS.Project - `stacks-cqrs-app`
+        - CQRS.Project - `stacks-cqrs`
```

Comment:
I really like this instead of having `app`, but unfortunately for now, we have to leave it as is due to the template name:
![image](https://github.com/user-attachments/assets/90715406-b037-4c25-9df2-cb03af1efaa6)